students/Sally_Shen/lesson7/html_render.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+    indent = "    "
+    def __init__(self, content=None, **attrs):
+        self.content = []
+        self.tag = "html"
+        if content:
+            self.content.append(content)
+        self.attrs = {}
+        if attrs:
+            self.attrs = attrs
+
+    def append(self, new_content):
+        self.content.append(new_content)
+        '''
```

Comment:
In general, when submitting code, you should try to remove out commented code which you are not using. This will just add clutter when it's not required